nss-cohort-40/banagzon-workforce-mgt-lumberjacks
# De department details

# Description
Added department details view in which a user can select a particular department from the list page, and then see a list of who works in that department.
 ## Type of change
 # Testing Instructions
Go to the department list page, click on any department link, and you should see a list of employees who work in that department or a message that says no one works there.
 # Checklist:
